src/canmatrix/cancluster.py

```diff
@@ -1,20 +1,20 @@
 # -*- coding: utf-8 -*-
 import typing
-import canmatrix
+import canmatrix.canmatrix as canmatrix
```

Comment:
CI seems py27 doesn't like this?

Reply:
It seems it's yet another problem :-(  Sorry for spamming with buildless commits, I'll probalby try to download py2 and try it locally.

Reply:
Tthe problem was missing `absolute_import` in cancluster.